privateca/snippets/src/test/java/privateca/SnippetsIT.java

```diff
@@ -346,6 +347,15 @@ public void testUpdateCertificateAuthority()
   public void testMonitorCertificateAuthority() throws IOException, InterruptedException {
     privateca.MonitorCertificateAuthority.createCaMonitoringPolicy(PROJECT_ID);
     assertThat(stdOut.toString()).contains("Monitoring policy successfully created !");
+
+    // cleanup created policy
```

Comment:
Would this fit best in [`cleanUp()`](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/34e41ee46a1647c19ee7556d0a6e6fa4fb74c571/privateca/snippets/src/test/java/privateca/SnippetsIT.java#L185)?

Reply:
It cannot because it is contextual for this particular test

Reply:
I see in other tests that we create resources in `setUp()` and then maintain a reference to these resources in an instance variable to be deleted later. See the use of `queue` [here](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/e97ca11ce87ffd28f3336c8fb675afb423baa6ff/tasks/snippets/src/test/java/com/example/task/ListQueuesIT.java#L44) or `CA_poolid` [here](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/e97ca11ce87ffd28f3336c8fb675afb423baa6ff/privateca/snippets/src/test/java/privateca/SnippetsIT.java#L66).

I wonder if this test should actually be pulled into the setup, with the new cleanup code moved into teardown. It works as-is, this is just something to consider.

Reply:
I think the use case is different. [SnippetIT][1] has many tests with only one that makes use of the specific example. There is no need to create such complex code to run it only once IMHO.

We can always refactor the solution when we will have more places that need to cleanup specific resources.

[1]: https://github.com/GoogleCloudPlatform/java-docs-samples/pull/8380/files#diff-995861e62924ab2481c923f27a70073a4d0284e26de93b111e7c966268b011dd